### DISCUSSION TITLE
Failing Unit Tests in LogEntryEventHandler_Tests and Logger_Tests classes

### DISCUSSION THREAD

**Discussion by dAzzl3** [jongpie/NebulaLogger#490] [Q&A]
Hi All,
I have noticed that since I have installed a unlocked package I got 3 unit test failing those are:
`LogEntryEventHandler_Tests.it_should_not_set_logging_user_as_log_owner_when_anonymous_mode_is_enabled, 
LogEntryEventHandler_Tests.it_should_set_logging_user_as_log_owner_when_anonymous_mode_is_disabled,
Logger_Tests.it_should_save_accurate_timestamp_when_logging_user_has_different_time_zone`.
They have common error `System.QueryException: List has no rows for assignment to SObject` while executing this query:
 `SELECT Id, TimeZoneSidKey FROM User WHERE (Name = 'Automated Process' AND Profile.Name = NULL)`.
In my org there is no User with Name "Automated Process" instead it is "Platform Integration User".
Is there suggestions how I can fix it?

Version of NabuleLogger -  4.9.10

Thanks

**Comment by jongpie** [comment]
Hi @dAzzl3 - those tests recently started failing a month or two ago, but they've been corrected [in the `v4.10.0` release](https://github.com/jongpie/NebulaLogger/releases/tag/v4.10.0). You can take a look the specific changes to fix these tests [in this commit](https://github.com/jongpie/NebulaLogger/pull/465/commits/425ac43b7a582f9c75dda68d030b6fdecf29f696). Hope this helps, let me know if you need anything else!
